src/styles/blocks/card.scss

```
@@ -0,0 +1,117 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition:
+    transform,
+    color,
+    300ms linear;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+    object-fit: cover;
+    display: block;
+  }
+
+  &__info {
+    margin-top: 40px;
+  }
+
+  &__title {
+    width: 100%;
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $card-main-color;
+
+    &:hover {
+      color: $card-hovered-text-color;
+    }
+  }
+
+  &__code {
+    margin: 0;
+    padding: 0;
+    margin-top: 4px;
+    color: $card-secondary-color;
+  }
+
+  &__review-section {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    margin: 0;
+    padding: 0;
+  }
+
+  &__price {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-top: 12px;
+  }
+
+  &__price-name {
+    margin: 0;
+    padding: 0;
+    font-size: 12px;
+    line-height: 18px;
+    color: $card-secondary-color;
+  }
+
+  &__price-amount {
+    margin: 0;
+    padding: 0;
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    text-align: right;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    margin: 0;
    padding: 0;
    display: flex;
    justify-content: space-between;
    width: 100%;
    margin-top: 12px;
    
    &-name {
      // write styles here
    }
    
    &-amount {
      // write styles here
    }
  }
```